### DISCUSSION TITLE
the game crashes during startupBug Report

### DISCUSSION THREAD

**Discussion by roki09** [FakeFishGames/Barotrauma#14387] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

the game crashes during startup

### Reproduction steps

1. Open game
2. Game crash
[crashreport (10).log](https://github.com/user-attachments/files/16462169/crashreport.10.log)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Duplicate of #14105. The issue is that the game cannot create the folders it needs:

> Cannot create directory at "C:\Users\user\AppData\Local\Daedalic Entertainment GmbH\Barotrauma\WorkshopMods\Installed": unauthorized access. The file/folder might be read-only! {Access to the path 'C:\Users\user\AppData\Local\Daedalic Entertainment GmbH\Barotrauma\WorkshopMods\Installed' is denied.}

Unless you've manually set some of the root folders to read-only, it's most likely due to an antivirus software, which is blocking the game.
